raft.c

```diff
@@ -881,13 +883,15 @@ static void handleLoadingState(RedisRaftCtx *rr)
         }
 
         initRaftLibrary(rr);
+        initSnapshot(rr);
```

Comment:
Changed this function's name as initSnapshotTransferData() to prevent confusion. It makes two things, explicitly sets outgoing snapshot mmap pointer to NULL and len to 0 and deletes if there is a partial incoming snapshot file.